### ISSUE TITLE
Readme / documentation for fmi2Simulate outdated

### ISSUE THREAD

**Issue by AnHeuermann** [ThummeTo/FMI.jl#170]
## Issue

Both the [README](https://github.com/ThummeTo/FMI.jl#readme) and the [Documentation](https://thummeto.github.io/FMI.jl/dev/library/#Simulate-FMUs) state that `fmi2Simulate` has two separate inputs for start and stop time:

```julia
fmiSimulate(str::fmi2Struct, t_start::Union{Real, Nothing} = nothing, t_stop::Union{Real, Nothing} = nothing;
            tolerance::Union{Real, Nothing} = nothing,
            dt::Union{Real, Nothing} = nothing,
            solver = nothing,
            customFx = nothing,
            recordValues::fmi2ValueReferenceFormat = nothing,
            saveat = [],
            setup::Bool = true,
            reset::Union{Bool, Nothing} = nothing, # nothing = auto
            inputValueReferences::fmi2ValueReferenceFormat = nothing,
            inputFunction = nothing,
            parameters::Union{Dict{<:Any, <:Any}, Nothing} = nothing,
            dtmax::Union{Real, Nothing} = nothing,
            kwargs...)
```

But I think in some of the latest versions of FMI.jl (v0.11.0 maybe?) the inputs `t_start` and `t_stop` became one tuple instead.

So the [example from the README](https://github.com/ThummeTo/FMI.jl#how-can-i-simulate-a-fmu-and-plot-values) doesn't work any more and the documentation isn't up to date.

## Versions 'n stuff

- FMI.jl @v0.11.3

**Comment by ThummeTo** [comment]
Jap, this is true, we are on this. Funny is we got that hint from another direction at the very same time :-D

**Comment by adribrune** [comment]
All right, it's in progress.

**Comment by ThummeTo** [comment]
@adribrune is this closed with one of the PRs the last two weeks?

**Comment by adribrune** [comment]
I just opened a PR for this issue.

**Comment by ThummeTo** [comment]
closed by #178
